Git-Change-the-URL-of-a-remote-repository.md

```diff
@@ -0,0 +1,23 @@
+The `git remote set-url` command changes an existing remote repository URL.
+
+The `git remote set-url` command takes two arguments:
```

Comment:
Just being picky about an otherwise nice write up. :wink: 

The second line is a bit repitative
> The `git remote set-url` command takes two arguments:

Can we change this to 
> It takes two arguments:
or 
> This takes two arguments: